packages/service-library/src/api-contracts/get-website-response.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { DocumentDataOnly, Website } from 'storage-documents';
+
+export interface GetWebsiteResponse extends DocumentDataOnly<Website> {
```

Comment:
Probably a bit confusing name for a type. We may use it across solution when working with data from a storage layer. Can we name it just `Website` and move under dedicated package to hold just REST API types?